app/uk/gov/hmrc/apidefinition/config/AppConfig.scala

```diff
@@ -31,22 +30,22 @@ class AppConfig @Inject() (val runModeConfiguration: Configuration, environment:
   lazy val buildProductionUrlForPrototypedAPIs: Boolean = runModeConfiguration.getOptional[Boolean]("buildProductionUrlForPrototypedAPIs").getOrElse(false)
   lazy val isSandbox: Boolean                           = runModeConfiguration.getOptional[Boolean]("isSandbox").getOrElse(false)
 
-  lazy val fetchByContextTtlInSeconds: String = runModeConfiguration.underlying.as[String]("fetchByContextTtlInSeconds")
+  lazy val fetchByContextTtlInSeconds: String = runModeConfiguration.underlying.getString("fetchByContextTtlInSeconds")
 
   lazy val ramlLoaderRewrites = buildRamlLoaderRewrites
 
   lazy val serviceBaseUrl = runModeConfiguration.getOptional[String]("serviceBaseUrl").getOrElse("http://localhost")
 
-  lazy val apisToRemove = runModeConfiguration.underlying.as[List[String]]("apisToRemove")
```

Comment:
These can be `get[Seq[String]]("apisToRemove").toList` for example. Just a bit shorted.  There is however no `get[List[String]]` doh.